cpu/cc2538/periph/spi.c

```diff
@@ -26,40 +26,44 @@
 #include "assert.h"
 #include "periph/spi.h"
 
+#include "vendor/hw_memmap.h"
+#include "vendor/hw_ssi.h"
+#include "cc2538_ssi.h"
+
 /**
  * @brief   Array holding one pre-initialized mutex for each SPI device
  */
 static mutex_t locks[SPI_NUMOF];
 
 static inline cc2538_ssi_t *dev(spi_t bus)
 {
-    return spi_config[bus].dev;
+    return (cc2538_ssi_t *)(SSI0_BASE + (spi_config[bus].num * 0x1000));
```

Comment:
This function is called in every `acquire`, `transfer`, `release` function. Do you have any estimation how this change effects performance?

Reply:
then it will be constant lookup just as before.